proposals/lip-0041.md

```diff
@@ -216,3 +295,5 @@ authAccount(senderAddress).nonce += 1
 ## Backwards Compatibility
 
 This LIP defines a new storage interface for the Auth module, which in turn will become part of the state tree and will be authenticated by the state root. An existing chain including the Auth module will need to perform a hardfork.
+
+[research:genesis-block]: https://research.lisk.com/t/update-genesis-block-schema-and-processing/325
```

Comment:
I don't see this link being used in the doc body, what was it expected to be used somwhere?

Reply:
I had a sentence to link the genesis block LIP but later removed so that we don't have to wait. I forgot to remove the link.